### ISSUE TITLE
Can't close modal dialogs in admin

### ISSUE THREAD

**Issue by isaacfreeman** [conversionfoundry/breeze#177]
When an admin clicks on the Cancel button for a modal dialog (e.g. the New Page dialog), the dialog doesn't close.

The line

```
$(this).dialog "close"
```

raises JS error

```
RangeError: Maximum call stack size exceeded.
```

**Comment by isaacfreeman** [comment]
This occurs when the dialog has been set to "modal: true". If the dialog is changed to non-modal, there's no problem.

**Comment by isaacfreeman** [comment]
The issue disappears if 

```
//= require twitter/bootstrap
```

is removed from breeze.js

**Comment by isaacfreeman** [comment]
Looks like the issue is with Bootstrap 2.3.0.

Inconveniently, the twitter-bootstrap-rails gem's version numbers aren't synced to Bootstrap's version numbers, and they went to 2.3.0 in version 2.2.2.

I'll fix the gem version for Breeze at 2.2.1.
